label_studio/io_storages/all_api.py

```diff
@@ -76,6 +80,8 @@ def get(self, request, **kwargs):
     name='get',
     decorator=swagger_auto_schema(
         tags=['Storage'],
+        x_fern_sdk_group_name='import_storage',
+        x_fern_sdk_method_name='list',
```

Comment:
should `projects` group be added here?

Reply:
it doesn't seem to work at all actually, only `/types` endpoints are working. Shall we remove these ones?
the `/storages/types` endpoint returns the list of globally available types of storage, unrelated to project. I'll leave them under `client.import_storage.list_types()`